pkg/controller/cainjector/sources.go

```diff
@@ -117,6 +117,13 @@ func (c *certificateDataSource) ReadCA(ctx context.Context, log logr.Logger, met
 		// don't requeue if we're just not found, we'll get called when the secret gets created
 		return nil, dropNotFound(err)
 	}
+	// Only use Secrets that have been created by this Certificate.
+	// The Secret must have a `cert-manager.io/certificate-name` annotation
+	// value matching the name of this Certificate..
```

Comment:
Thanks again for adding these bits of information! Nit:

```suggestion
	// Only use Secrets that have been created by this Certificate.
	// The Secret must have a `cert-manager.io/certificate-name` annotation
	// value matching the name of this Certificate.
```